crates/oxc_linter/src/utils/jest/parse_jest_fn.rs

```diff
@@ -100,7 +101,8 @@ pub fn parse_jest_fn_call<'a>(
 
         let mut call_chains = Vec::from([Cow::Borrowed(name)]);
         call_chains.extend(members.iter().filter_map(KnownMemberExpressionProperty::name));
-        if !is_valid_jest_call(&call_chains) {
+
+        if !is_valid_jest_call(&call_chains) && !is_valid_vitest_call(&call_chains) {
```

Comment:
Seems like this should be:

```js
if ctx.frameworks().is_jest() && !is_valid_jest_call(&call_chains) {
    return None;
}

if ctx.frameworks().is_vitest() && !is_valid_vitest_call(&call_chains) {
    return None;
}
```

After changing this, it will also expose some bugs in our jest rules. Are you interest in fixing them in another PR?

Reply:
Sure, but I'm not sure where the bug is. 😅

I ran the `cargo test` and test cases are passed. 

Reply:
We need to update this first:

![CleanShot 2024-08-03 at 09 46 04@2x](https://github.com/user-attachments/assets/31a08f3f-816e-43ba-968e-3e2a8e6798be)

```
pub const fn is_jest(self) -> bool {
    self.contains(Self::Jest)
}
```

Reply:
Updated